collector/index.md

```diff
@@ -0,0 +1,425 @@
+---
+title: 'pganalyze Collector'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+The pganalyze collector periodically queries your database and sends metrics and metadata (as "snapshots") to
+the pganalyze app. This is the only component that talks directly to your database. For details on setting up
+the collector, see the [collector installation guide](/docs/install/02_installing_the_collector). The collector
+is open-source and the code is available [on GitHub](github.com/pganalyze/collector).
+
+
+# Configuration Settings
+
+The collector can be configured either through a `INI` config file (in a normal install) or environment variables
+(typically for running via its Docker image). Most settings can be configured through either mechanism (if both
+are present, the config file takes precedence). Note that a single collector instance can monitor more than one
+database server (though this is not supported when configuring through environment variables). The `[pganalyze]`
```

Comment:
(though this is not supported when configuring through environment variables)

=> `This is not supported when configuring through environment variables.`

Reply:
And make a new paragraph after this line.

Reply:
Ended up just dropping the parentheses in the first comment, and adding a comma--I think that reads better and makes it clearer what the "this" is.